### PR TITLE
Team stats refactor

### PR DESCRIPTION
**Pull request by gabichuelas**
@rrabinovitch @evilaspaas1 @sciencefixion 

Refactored my other team stats methods using my newfound knowledge (thanks Dan)

### CODE REVIEW

**Review by sciencefixion — APPROVED** [review]
Great work here! Glad it's all coming more easily. :)

**Comment by gabichuelas** [comment]
Created a new module, `TeamStats` to help with refactoring my Team methods. I basically put all my helper methods in there in order to take them out of the StatTracker file bc it's too big.

File: lib/game.rb [hunk 1]
```diff
@@ -17,4 +17,38 @@ def initialize(game_params)
     @total_goals = @away_goals.to_i + @home_goals.to_i
   end
 
+  def result
```
Comment (rrabinovitch) [inline lib/game.rb:20]: oh my god, so simple and so genius

File: lib/team.rb [hunk 2]
```diff
@@ -15,11 +15,11 @@ def initialize(team_params)
 
   def info
     {
-      team_id: @team_id,
-      franchise_id: @franchise_id,
-      team_name: @team_name,
-      abbreviation: @abbreviation,
-      link: @link
+      "team_id" => @team_id,
```
Comment (rrabinovitch) [inline lib/team.rb:18]: thanks for addressing these!

**Review by rrabinovitch — APPROVED** [review]

File: lib/team.rb (hunk 2 above)
Comment (rrabinovitch) [reply lib/team.rb:18]: oh wait, I'm realizing this isn't referring to the headers - whoops :)
still good call on this method!

**Pull request merged**